lib/utils/hookFiles.js

```diff
@@ -0,0 +1,48 @@
+import pathModule from "path"
+import fs from "fs"
+
+const createHookFiles = ({ path, hookName, dumbString, indexString, options }) => {
+	const hooksPath = pathModule.resolve(path, "hooks")
+	const dumbhookPath = pathModule.resolve(hooksPath, `${hookName}.jsx`)
+	const styleSheetPath = pathModule.resolve(hooksPath, options.scss ? `${hookName}.scss` : `${hookName}.css`)
```

Comment:
Just trolling, but you write `dumbhook` with no camelCase, then `styleSheet` using camelCase 😛

Reply:
Nice catch ;)